synapse/lib/stormtypes.py

```diff
@@ -3801,64 +3814,54 @@ def getObjLocals(self):
 
     async def _methQueueCull(self, offs):
         offs = await toint(offs)
-        gatekeys = self._getGateKeys('get')
-        await self.runt.reqGateKeys(gatekeys)
-        await self.runt.view.core.coreQueueCull(self.name, offs)
+        self.runt.confirm(('queue', 'get', self.iden), gateiden=self.iden)
```

Comment:
Don't need to duplicate the iden in the perm
```suggestion
        self.runt.confirm(('queue', 'get'), gateiden=self.iden)
```